When : when you try to use enable with libmaxmind, libpreludedb, xml, libwrap, they are not working
Why : Needed if you want to use useflags
--- a/configure.in
+++ b/configure.in
@@ -110,7 +110,10 @@
 dnl **************************************************
 LIBMAXMINDDB_MIN_VERSION=1.0.0
 
-PKG_CHECK_MODULES([LIBMAXMINDDB], [libmaxminddb >= $LIBMAXMINDDB_MIN_VERSION], enable_libmaxminddb=yes, enable_libmaxminddb=no)
+AC_ARG_ENABLE(libmaxminddb, AC_HELP_STRING(--enable-libmaxminddb, Define whether libmaxminddb is available), , enable_libmaxminddb="yes")
+if test x$enable_libmaxminddb = xyes; then
+	PKG_CHECK_MODULES([LIBMAXMINDDB], [libmaxminddb >= $LIBMAXMINDDB_MIN_VERSION], enable_libmaxminddb=yes, enable_libmaxminddb=no)
+fi
 if test x$enable_libmaxminddb = xyes; then
 	AC_CHECK_HEADER(maxminddb.h, enable_libmaxminddb=yes, enable_libmaxminddb=no)
 fi
@@ -126,11 +129,14 @@
 dnl * Check for libpreludedb                         *
 dnl **************************************************
 
-AM_PATH_LIBPRELUDEDB(3.0.0, enable_libpreludedb=yes, enable_libpreludedb=no, no)
-AM_CONDITIONAL(HAVE_LIBPRELUDEDB, test x$enable_libpreludedb = xyes)
+AC_ARG_ENABLE(libpreludedb, AC_HELP_STRING(--enable-libpreludedb, Define whether libpreludedb is available), , enable_libpreludedb="yes")
+if test x$enable_libpreludedb = xyes; then
+	AM_PATH_LIBPRELUDEDB(3.0.0, enable_libpreludedb=yes, enable_libpreludedb=no, no)
+fi
 if test x$enable_libpreludedb = xyes; then
         AC_DEFINE_UNQUOTED(HAVE_LIBPRELUDEDB, , Define whether libpreludedb is available)
 fi
+AM_CONDITIONAL(HAVE_LIBPRELUDEDB, test x$enable_libpreludedb = xyes)
 
 
 
@@ -138,7 +144,10 @@
 dnl * Check for the Libxml2                                *
 dnl ********************************************************
 
-AM_PATH_XML2(, enable_xmlmod=yes, enable_xmlmod=no)
+AC_ARG_ENABLE(xmlmod, AC_HELP_STRING(--enable-xmlmod, Enable XML), , enable_xmlmod="yes")
+if test x$enable_xmlmod = xyes; then
+	AM_PATH_XML2(, enable_xmlmod=yes, enable_xmlmod=no)
+fi
 AM_CONDITIONAL(HAVE_XML2, test x$enable_xmlmod = xyes)
 
 
@@ -147,9 +156,10 @@
 dnl * TCP WRAPPER CHECK                              *
 dnl **************************************************
 
+AC_ARG_WITH(libwrap, AC_HELP_STRING(--with-libwrap, Compile in libwrap (tcp_wrappers) support.), , with_libwrap="yes")
 AC_ARG_WITH(libwrap-prefix, AC_HELP_STRING(--with-libwrap-prefix@<:@=PFX@:>@,
                                     Compile in libwrap (tcp_wrappers) support @<:@default=auto@:>@.),
-            libwrap_required=true, with_libwrap="yes")
+            libwrap_required=true, )
 
 if test x$with_libwrap != xno; then
    LIBWRAP_INCLUDE="tcpd.h"
